certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/validation/model/ElementValidation.java

```diff
@@ -7,5 +7,6 @@
 
 public interface ElementValidation {
 
-  List<ValidationError> validate(ElementData data, Optional<ElementId> categoryId);
-}
+  List<ValidationError> validate(ElementData data, Optional<ElementId> categoryId,
+      List<ElementData> elementData);
```

Comment:
Maybe change name of elementData to dataList to match the first argument that is data. Whatever name decided make sure that the implementations reflect the name.